### ISSUE TITLE
Confusing display of text in lists

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2812]
When I saw this

![image](https://user-images.githubusercontent.com/2138696/111622725-7a7acb00-8824-11eb-9724-fa42c3fde0d4.png)

I was concerned that the first element was " Text" rather than "Text". But it is "Text" yet it is displayed in a misleading way.

**Comment by DarDoro** [comment]
![Untitled script pic (80)](https://user-images.githubusercontent.com/16109071/111857574-b9974200-8932-11eb-8263-127fb9e32f18.png)

**Comment by Rexxt** [comment]
It appears the displays are centered, but their lengths vary depending on the length of the text...

**Comment by ToonTalk** [comment]
Yes, but is that a good idea? Looking at my list I think it is too easy to think that the first element is " Text" instead of "Text". Left justified text would not have this problem.

**Comment by ToonTalk** [comment]
Consider this
![image](https://user-images.githubusercontent.com/2138696/115048395-18d57b80-9f0c-11eb-99df-30c918f9b609.png)

**Comment by berrymot** [comment]
i see how that could be confusing. perhaps there's a way to have the visible space thing in "shown" variable / list / reporter things?
